src/components/common/Interactive.tsx

```diff
@@ -40,55 +26,35 @@ interface Props {
 
 const InteractiveBase = ({ onMove, onKey, ...rest }: Props) => {
   const container = useRef<HTMLDivElement>(null);
-  const hasTouched = useRef(false);
-  const [isDragging, setDragging] = useState(false);
```

Comment:
Don't need it anymore, which improves the performance

Reply:
Nice!